cmd/common.go

```diff
@@ -136,6 +147,18 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 		}
 	}
 
+	//Print summary Level-wise
```

Comment:
As mentioned in the conversation, I don't think we want to 
- output a level 2 summary if we're only running level 1 tests 
- output a level 1 summary that's going to be identical to the overall summary